app/models/concerns/hydranorth/generic_file/metadata.rb

```diff
@@ -81,7 +81,7 @@ module Metadata
         property :remote_resource, predicate: ::UALTerms.remote_resource, multiple:false
 
         property :doi, predicate: ::UALId.doi, multiple: false do |index|
-          index.as :stored_searchable
+          index.as :symbol
```

Comment:
I tried using  `index.as :symbol, :stored_searchable` as this  what I found in other properties we are making a symbol (hasCollectionID, belongsToCommunity, hasCollection, fedora3uuid, fedora3handle, etc, etc). But this does not get what we want. Matt discovered, 

Symbol definition:
https://github.com/projecthydra/solrizer/blob/master/lib/solrizer/default_descriptors.rb#L35

Stored_searchable definition:
https://github.com/projecthydra/solrizer/blob/master/lib/solrizer/default_descriptors.rb#L80

You will see on line 80 of the above link, it will take anything that is a string/text and cast it into a text_en which doesn't allow us to search and match whole words.